naga/src/front/wgsl/parse/directive/enable_extension.rs

```diff
@@ -121,12 +124,19 @@ pub enum UnimplementedEnableExtension {
     ///
     /// [`enable clip_distances;`]: https://www.w3.org/TR/WGSL/#extension-clip_distances
     ClipDistances,
+    /// Enables subgroup built-ins in all languages.
```

Comment:
todo: I'm not sure how accurate this statement is yet.

Reply:
I'm not entirely sure what it means 😅

Reply:
This is referring to language frontends. I see that SPIR-V input also parses subgroups, so I think this is fine.

Reply:
In Naga's concept of the world, only sort of. 🙂 Once something enters IR like "is this extension available?", it's no longer _just_ a frontend thing; other front ends will need to emit IR with this enable extension set, likely when the front end detects that it's being used (rather than explicitly negotiated like in WGSL).